VERSIONS.md

```diff
@@ -0,0 +1,14 @@
+| Name           | Numeric | Ruby  | Rails | PostgreSQL | Kafka    |
+| -------------- | ------- | ----- | ----- | ---------- | -------- |
+|                |         |       |       |            |          |
+| Anand          |  1.y.z  |       |       |            |          |
+| Botvinnik      |  2.y.z  |       |       |            |          |
+| Capablanca     |  3.y.z  | 2.2.z | 4.2.z | 9.4.z      |          |
+| Darga          |  4.y.z  | 2.2.z | 5.0.z | 9.4.z      |          |
+| Euwe           |  5.y.z  | 2.3.z | 5.0.z | 9.5.z      |          |
+| Fine           |  6.y.z  | 2.3.z | 5.0.z | 9.5.z      |          |
+| Gaprindashvili |  7.y.z  | 2.3.z | 5.0.z | 9.5.z      |          |
+| Hammer         |  8.y.z  | 2.4.z | 5.0.z | 9.5.z      |          |
+| Ivanchuk       |  9.y.z  | 2.5.z | 5.1.z | 10.y       |          |
+| Jansa          | 10.y.z  |       |       |            | optional |
+| Kasparov       | 11.y.z  |       |       |            | optional |
```

Comment:
@bdunne @Fryguy @simaishi can you help with the versions of Ruby, Rails and PostgreSQL in Jansa and Kasparov?

Reply:
Got the info and updated the table